documentation/docs/pages/operators/nodes/nym-node/_meta.json

```diff
@@ -1,5 +1,11 @@
 {
```

Comment:
I can't render the operator guides, looks like an unincluded component?

Reply:
Did you install accordion?